src/js/views/patients/schedule/schedule_views.js

```diff
@@ -116,15 +116,18 @@ const DayItemView = View.extend({
   templateContext() {
     const state = this.model.getState();
 
+    const isReduced = this.state.get('isReduced');
+
     return {
       isOverdue: this.model.isOverdue(),
       state: state.get('name'),
       stateOptions: state.get('options'),
       patient: this.model.getPatient().attributes,
       form: this.model.getForm(),
-      isSelected: this.state.isSelected(this.model),
+      isSelected: !isReduced && this.state.isSelected(this.model),
```

Comment:
When the `stateModel` is removed from the `reduced_schedule_app.js` file, this line of code causes an error when the current clinician is a reduced schedule user. It looks for the `this.state.isSelected()` function but can’t find it since it usually comes from the `StateModel` in the `schedule_state.js` file.

Adding `!isReduced` stops the `this.state.isSelected()` function from being called when the view is in the `isReduced` state (when the `StateModel` isn’t present). And keeps it functioning the same in the normal, non-reduced state.

Reply:
Seems like a reasonable solution for this mini-fork.  I hope we don't regret the simplification later, but works great for now.

Reply:
Why would we regret it in the future? Out of curiosity.